content/en/tracing/compatibility_requirements/dotnet-core.md

```diff
@@ -10,7 +10,7 @@ further_reading:
 
 The .NET Datadog Trace library is open source - view the [Github repository][1] for more information.
 
-The .NET Tracer supports automatic instrumentation on .NET Core 2.1 and 3.1. It also supports [.NET Framework][2].
+The .NET Tracer supports automatic instrumentation on .NET 5 (starting with .NET Tracer `1.20.0`) and on LTS .NET Core versions 2.1 and 3.1. It also supports [.NET Framework][2].
```

Comment:
I don't think we generally mention specific tracer versions in the docs (it's implied that everything applies to the _current_ version), and LTS may be confusing for users ("is that a special non-standard version of .NET Core?"). Maybe we should keep it simple, like:

```suggestion
The .NET Tracer supports automatic instrumentation on .NET 5, .NET Core 3.1, and .NET Core 2.1. It also supports [.NET Framework][2].
```